app/controllers/proposed_organisations_controller.rb

```diff
@@ -33,8 +33,8 @@ def create
       send_email_to_superadmin_about_org_signup @proposed_organisation
       redirect_to @proposed_organisation, notice: 'Organisation is pending admin approval.'
     else
-      flash[:error] = @proposed_organisation.errors.full_messages.join('<br/>').html_safe
-      redirect_to new_proposed_organisation_path and return false
+      @categories_start_with = Category.first_category_name_in_each_type
```

Comment:
@johnnymo87 in case of error we should render the form ( which requires this variable `@categories_start_with` to list different categories)